sections/basics/styling-any-components.js

```diff
@@ -4,7 +4,7 @@ const StylingAnyComponents = () => md`
   ## Styling any components
 
   The \`styled\` method works perfectly on all of your own or any third-party
-  components as well, as long as they're accepting the \`className\` prop.
+  components as well, as long as they pass the \`className\` prop to their rendered sub-components.
```

Comment:
This is not entirely accurate, the className just has to be attached to a DOM node!

Reply:
Okay. Just to be sure I understand - given that component A renders component B which in turn renders an actual DOM node (say, a `div`) - it's *not* enough to styled(A) to pass the `className` to B, but B also has to pass it down to the `div`?